app/modules/intelligence/tools/get_workspace_debug_context_tool.py

```
@@ -0,0 +1,255 @@
+                InferredCommand(**ic) for ic in (result.get("inferred_commands") or [])
+            ]
+            return WorkspaceDebugContext(
+                launch_configs=launch_configs,
+                debug_adapters=debug_adapters,
+                recent_changes=recent_changes,
+                related_tests=related_tests,
+                inferred_commands=inferred_commands,
+                available=True,
+                message=None,
+            ).model_dump()
+        except Exception as parse_exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)